### ISSUE TITLE
Adding ruby linting to the CI checks to ensure styling is consistent

### ISSUE THREAD

**Issue by edwinthinks** [rubyforgood/shelter-assist#113]
I'am wondering if we want to utilize some linting library like https://github.com/rubocop/rubocop or https://github.com/testdouble/standard? 

This can help keep our code consistent.

**Comment by edwinthinks** [comment]
@freestylebit @chriszimmerman thoughts?

**Comment by chriszimmerman** [comment]
I'm using rubocop at my current job and I love it. Using an automated linter gets rid of a class of problems so I'm in full support of adding one. I think it would be valuable to have a JS one as well.

**Comment by freestylebit** [comment]
Yep, let's do this!  I used rubocop previously with good results, too!
